### PR TITLE
chore: support exact version of packages during release

### PR DESCRIPTION
**Pull request by tolerants**
- Support exact version of packages during release
- Added ssr build to the testfront

closes: [HRZ-3145](https://spryker.atlassian.net/browse/HRZ-3145), [HRZ-3195](https://spryker.atlassian.net/browse/HRZ-3195)

[HRZ-3145]: https://spryker.atlassian.net/browse/HRZ-3145?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

[HRZ-3195]: https://spryker.atlassian.net/browse/HRZ-3195?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by tobi-or-not-tobi — DISMISSED** [review]

File: libs/template/application/server/src/server.prod.ts [hunk 1]
```diff
@@ -31,7 +31,6 @@ export async function createProdSever(
 
       res.status(200).end(html);
     } catch (e) {
-      console.error(e);
```
Comment (AndreyYevtukhov) [inline libs/template/application/server/src/server.prod.ts]: Do we have to remove it from here?
This error appears in Netlify function logs and might be useful for debugging

File: apps/testfront/package.json [hunk 2]
```diff
@@ -6,11 +6,11 @@
...
   "scripts": {
     "dev": "vite -c vite.config.dev.ts",
-    "dev:ssr": "ts-node --experimentalSpecifierResolution node --esm server/server",
-    "build": "npm run build:client && npm run build:ssr && npm run build:netlify",
+    "dev:ssr": "ts-node server/server",
+    "prod:ssr": "npm run build && npm run dev:ssr --omit=dev",
+    "build": "npm run build:client && npm run build:ssr",
     "build:client": "vite build",
-    "build:ssr": "vite build -c vite.config.server.ts",
-    "build:netlify": "vite build -c vite.config.netlify.ts"
```
Comment (AndreyYevtukhov) [inline apps/testfront/package.json:13]: Did we drop this file? (It seems like it is not needed anymore)

File: libs/template/application/server/src/server.prod.ts (hunk 1 above)
Comment (tolerants) [reply libs/template/application/server/src/server.prod.ts]: Moved back, thanks.

File: apps/testfront/package.json (hunk 2 above)
Comment (tolerants) [reply apps/testfront/package.json:13]: Yep, removed it for now.

**Review by AndreyYevtukhov — APPROVED** [review]

**Pull request merged**